src/helpers/graphqlHelpers.ts

```diff
@@ -48,55 +82,31 @@ const parseGraphqlGetRequest = (
     (qs) => qs.name === "extensions"
   )
 
-  try {
-    const variables =
-      variablesParam?.value && decodeURIComponent(variablesParam?.value)
-
-    if (queryParam?.value) {
-      try {
-        const query = decodeURIComponent(queryParam.value)
-
-        if (variables) {
-          return [
-            {
-              query,
-              variables: JSON.parse(variables),
-            },
-          ]
-        }
-
-        return [
-          {
-            query,
-          },
-        ]
-      } catch (e) {
-        return null
-      }
-    }
+  const query = queryParam && decodeQueryParam(queryParam.value)
```

Comment:
If queryParam is the number 0 query will compute to 0. Could we wrap queryParam in Boolean, or exit early in `decodeQueryParam` if invalid.

`const query = queryParam ? decodeQueryParam(queryParam.value) : null`

Reply:
`queryParam` is either an object or undefined. I am simply checking the object is not undefined here.